app/views/options-on-interstitial/check-state-pension-1.html

```diff
@@ -0,0 +1,118 @@
+{% extends "layout.html" %}
+
+{% block page_title %}
+  Start page example
+{% endblock %}
+
+{% block proposition_header %}
+  <!-- blank to remove the proposition header -->
+{% endblock %}
+
+{% block header_class %}
+  <!-- blank to remove the proposition header -->
+{% endblock %}
+
+{% block content %}
+
+<main id="content" role="main">
+
+  <div class="breadcrumbs">
+    <ol>
+      <li><a href="#">Home</a></li>
+      <li><a href="#">Working, jobs and pensions</a></li>
+      <li><a href="#">State Pension</a></li>
+    </ol>
+  </div>
+
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <h1 class="heading-xlarge">
+        {% if serviceName %} {{ serviceName }} {% endif %}
+      </h1>
+
+      <p>Use this service to find out:</p>
+
+      <ul class="list list-bullet">
+        <li>how much State Pension you could get (this amount is also known as your State Pension forecast)</li>
+        <li>when you can get it</li>
+        <li>how to increase it, if you can</li>
+      </ul>
+
+      <div role="note" class="panel panel-border-wide">
+        <p>
+          This page is also available in Welsh (<a href="#">Cymraeg</a>).
+        </p>
+      </div>
+      <p>
+        <a href="https://www.gov.uk/government/news/proposed-new-timetable-for-state-pension-age-increases">The State Pension age is under review</a>
+        and may change in the future.
+      </p>
+      <div role="note" class="panel panel-border-wide">
+        <p>
+          You can’t use this service if you’re already getting your State Pension or if you’ve delayed (‘deferred’) claiming it.
+        </p>
+      </div>
+
+      <h2 class="heading-medium">You need to sign in</h2>
+      <p>
+        Lorem ipsum dolor sit amet, te vix nisl splendide signiferumque, mel reque appetere cu. Pro eu populo conceptam. Id copiosae dissentias eam. Eum ea dico exerci dicunt, mea te habeo.
+      </p>
+      <a class="button" type="submit" href="./check-state-pension-interstitial-1">Sign in</a>
```

Comment:
type="submit" is used on the `<input>` tag, here we could use `role="button"` instead to indicate that these are buttons.

Reply:
I've changed this to `role="button"` 👍